equistore/build.rs

```diff
@@ -46,15 +46,28 @@ fn main() {
         equistore_core.push(splitted[..splitted.len() - 1].join("."));
     }
 
-    let mut install_dir = cmake::Config::new(&equistore_core)
+    let install_dir = cmake::Config::new(&equistore_core)
         .define("CARGO_EXE", env!("CARGO"))
         .define("RUST_BUILD_TARGET", std::env::var("TARGET").unwrap())
+        .define("BUILD_SHARED_LIBS", if cfg!(feature="static") { "OFF" } else { "ON" })
+        .define("EQUISTORE_INSTALL_BOTH_STATIC_SHARED", "OFF")
```

Comment:
@PicoCentauri this should be our guard against this failure: we don't install the "other" library anymore, so if we break this again it will fail to link.

Reply:
Ahh yes I see. Okay lets go for it then.